R/pkg/R/stats.R

```diff
@@ -19,7 +19,8 @@
 
 setOldClass("jobj")
 
-#' crosstab
+#' @title SparkDataFrame statistic functions
```

Comment:
For me, the generated html file, `file:///Users/dongjoon/spark/R/pkg/html/statfunctions.html` has the following title.
```
SparkDataFrame statistic functions crosstab - Computes a pair-wise frequency table of the given columns Computes a pair-wise frequency table of the given columns. Also known as a contingency table. The number of distinct values for each column should be less than 1e4. At most 1e6 non-zero pair frequencies will be returned.
```
Also, `index.html` shows the above long string for all the stat functions like `approxQuantile`.